pkg/descheduler/strategies/pod_antiaffinity_test.go

```diff
@@ -111,21 +159,28 @@ func TestPodAntiAffinity(t *testing.T) {
 			return true, &v1.PodList{Items: test.pods}, nil
 		})
 		fakeClient.Fake.AddReactor("get", "nodes", func(action core.Action) (bool, runtime.Object, error) {
-			return true, node, nil
+			return true, node1, nil
 		})
 
+		nodes := test.nodes
+
 		podEvictor := evictions.NewPodEvictor(
 			fakeClient,
 			policyv1.SchemeGroupVersion.String(),
 			false,
 			test.maxPodsToEvictPerNode,
-			[]*v1.Node{node},
+			nodes,
```

Comment:
Why not `test.nodes`? I don't see `nodes` used anywhere besides here and in `RemovePodsViolatingInterPodAntiAffinity`

Reply:
I had that a few lines above, but it looks like I forgot to make it inline. It's there now.